src/test/java/net/sf/jsqlparser/statement/delete/DeleteTest.java

```diff
@@ -118,8 +123,19 @@ public void testWith() throws JSQLParserException {
                 + "DELETE FROM cfe.instrument_ref\n"
                 + "WHERE  id_instrument_ref = (SELECT id_instrument_ref\n"
                 + "                            FROM   a)";
-
         assertSqlCanBeParsedAndDeparsed(statement, true);
```

Comment:
Please note that `assertSqlCanBeParsedAndDeparsed` returns the parsed `Statement` and so you can simplify to

```java
Delete delete = (Delete)  assertSqlCanBeParsedAndDeparsed( sqlStr, true );
```

which saves you one parse step.

Please apply to all your changed tests.

Reply:
Thanks @manticore-projects - Ive just pushed [this commit](https://github.com/JSQLParser/JSqlParser/pull/2051/commits/0566fb0883fd923ef571e9ce4e132b0f38648aa1) to address this